### ISSUE TITLE
Define parameters for `plotly()`

### ISSUE THREAD

**Issue by sckott** [ropensci/plotly#206]
Maybe i'm missing something, but could parameters defs be added for the parameters in a call to `x$ggplotly()` ?  eg.. when I look at the function definition, 

```r
function (gg = last_plot(), kwargs = list(filename = NULL, fileopt = NULL, 
    width = NULL, height = NULL), session = "interactive") 
...
```

e.g., `gg`, `kwargs`, and `session` don't seem to be documented

a user commented about this in the guest blog post on our site https://ropensci.org/blog/2014/04/17/plotly/#comment-1971263190

**Comment by mkcor** [comment]
@sckott They are not documented in a docstring indeed. There are several examples around the web... But I agree this should be part of the package. How would you go about writing a docstring when the function is a method living under `plotly <- function(username=NULL, key=NULL, base_url=NULL) { ... }`?

**Comment by sckott** [comment]
hmm, not sure. Though the way you set up the function, [R6](https://github.com/wch/R6) seems like a perfect fit . It sounds like there isn't yet, but will be soon support for documenting R6 in roxygen2, eg., see  https://github.com/klutometis/roxygen/issues/306

There are some workaround discussed in the `R6` issues here https://github.com/wch/R6/issues/3

**Comment by cpsievert** [comment]
Fixed as of version 1.0

**Comment by sckott** [comment]
:+1:
